charts/alfresco-connector-hxi/templates/deployment-connector-hxi-live-ingester.yaml

```
@@ -0,0 +1,69 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "alfresco-connector-hxi.live-ingester.fullname" . }}
+  labels:
+    {{- include "alfresco-connector-hxi.live-ingester.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.liveIngester.replicaCount | default 1 }}
+  selector:
+    matchLabels:
+      {{- include "alfresco-connector-hxi.live-ingester.selectorLabels" . | nindent 6 }}
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      {{- toYaml .Values.liveIngester.strategy.rollingUpdate | nindent 6 }}
+  template:
+    metadata:
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/configmap-connector-hxi.yaml") $ | sha256sum }}
+        checksum/secret: {{ include (print $.Template.BasePath "/secret-connector-hxi.yaml") $ | sha256sum }}
+        checksum/secret-broker: {{ include (print $.Template.BasePath "/secret-mq.yaml") $ | sha256sum }}
+        {{- with $.Values.podAnnotations }}
+          {{- toYaml $ | nindent 8 }}
+        {{- end }}
+      labels:
+        {{- include "alfresco-connector-hxi.live-ingester.labels" . | nindent 8 }}
+    spec:
+      {{- include "alfresco-common.imagePullSecrets" . | indent 6 }}
+      serviceAccountName: {{ include "alfresco-connector-hxi.serviceAccountName" . }}
+      {{- include "alfresco-common.component-pod-security-context" .Values | indent 4 }}
+      containers:
+        - name: {{ include "alfresco-connector-hxi.live-ingester.name" . }}
+          image: "{{ .Values.liveIngester.image.repository }}:{{ .Values.liveIngester.image.tag }}"
+          imagePullPolicy: {{ .Values.liveIngester.image.pullPolicy }}
+          {{- include "alfresco-common.component-security-context" .Values | indent 8 }}
+          env:
+            {{- include "alfresco-connector-hxi.activemq.cm.env" $ | indent 12 }}
+            {{- include "alfresco-connector-hxi.activemq.secret.env" $ | indent 12 }}
+            {{- include "alfresco-connector-hxi.sfs.cm.env" $ | indent 12 }}
+            {{- include "alfresco-connector-hxi.repository.cm.env" $ | indent 12 }}
+            {{- include "alfresco-connector-hxi.cm.env" $ | indent 12 }}
+            {{- include "alfresco-connector-hxi.secret.env" $ | indent 12 }}
+            {{- range $key, $val := $.Values.liveIngester.environment }}
+            - name: {{ $key }}
+              value: {{ $val | quote }}
+            {{- end }}
+          ports:
+            - containerPort: {{ .Values.liveIngester.image.internalPort }}
+          resources: {{- toYaml .Values.resources | nindent 12 }}
+          livenessProbe:
+            initialDelaySeconds: 30
+            periodSeconds: 10
+            timeoutSeconds: 3
+            httpGet:
+              path: /actuator/health/liveness
+              port: 8080
```

have it tunable via values 

```suggestion
            {{- toYaml .Values.liveIngester.livenessProbe | nindent 12 }}
```